src/families/ethereum/synchronisation.ts

```diff
@@ -159,6 +178,7 @@ export const getAccountShape: GetAccountShape = async (
     lastSyncDate: new Date(),
     balanceHistory: undefined,
     syncHash,
+    NFT,
```

Comment:
naming would be a bit off keep it camel case if you can.

Reply:
Ok will do, I just followed the example given by @gre a month ago 👍